src/go/k8s/controllers/redpanda/cluster_controller.go

```diff
@@ -180,7 +180,9 @@ func (r *ClusterReconciler) Reconcile(
 		pki.AdminAPINodeCert(),
```

Comment:
oh my god, this is terrible. :D not your fault at all but it's terrible

Reply:
😅